modules/scalacheck/src/weaver/scalacheck/Checkers.scala

```diff
@@ -22,32 +24,32 @@ trait Checkers {
   // Configuration for property-based tests
   def checkConfig: CheckConfig = CheckConfig.default
 
+  private implicit val genMonad: Monad[Gen] = new StackSafeMonad[Gen] {
+    override def flatMap[A, B](fa: Gen[A])(f: A => Gen[B]): Gen[B] =
+      fa.flatMap(f)
+
+    def pure[A](x: A): Gen[A] = Gen.const(x)
+  }
+
   class PartiallyAppliedForall(config: CheckConfig) {
 
     def apply[A1: Arbitrary: Show, B: PropF](f: A1 => B)(
         implicit loc: SourceLocation): F[Expectations] =
       forall(implicitly[Arbitrary[A1]].arbitrary)(liftProp[A1, B](f))
 
-    def apply[A1: Arbitrary: Show, A2: Arbitrary: Show, B: PropF](f: (
-        A1,
-        A2) => B)(
+    def apply[A1: Arbitrary: Show, A2: Arbitrary: Show, B: PropF](
+        f: (A1, A2) => B)(
         implicit loc: SourceLocation): F[Expectations] =
-      forall(implicitly[Arbitrary[(A1, A2)]].arbitrary)(liftProp(
-        f.tupled))
+      forall(implicitly[Arbitrary[(A1, A2)]].arbitrary)(liftProp(f.tupled))
 
     def apply[
         A1: Arbitrary: Show,
         A2: Arbitrary: Show,
         A3: Arbitrary: Show,
         B: PropF](
         f: (A1, A2, A3) => B)(
-        implicit loc: SourceLocation): F[Expectations] = {
-      implicit val tuple3Show: Show[(A1, A2, A3)] = {
-        case (a1, a2, a3) => s"(${a1.show},${a2.show},${a3.show})"
-      }
```

Comment:
**thought:** removed these Shows as Cats now has them built in. It appears that they work in the same way.